### ISSUE TITLE
Should not allow reserved words for variables which are declared by import declarations

### ISSUE THREAD

**Issue by mysticatea** [eslint/espree#225]
```js
import {class, default, var} from "foo";
```

Currently, espree succeeds to parse this code.
But should raise a syntax error.

http://tc39.github.io/ecma262/#prod-ImportSpecifier

```
ImportSpecifier:
    ImportedBinding
    IdentifierName "as" ImportedBinding

ImportedBinding:
    BindingIdentifier

BindingIdentifier[Yield]:
    Identifier
    [~Yield] "yield"

Identifier::
    IdentifierName but not ReservedWord
```

Reserved words are allowed only where followed by "as".

**Comment by nzakas** [comment]
What version are you using?

**Comment by mysticatea** [comment]
`3.0.0-alpha-3` in my PC and the online demo.

**Comment by nzakas** [comment]
Then most likely this is an Acorn bug.

**Comment by nzakas** [comment]
Related: https://github.com/ternjs/acorn/issues/365
